### DISCUSSION TITLE
Stake option not working

### DISCUSSION THREAD

**Discussion by TJ-BitCanna** [BitCannaGlobal/WeedMobile#83] [General]
As mentioned by @JohnnyX89; "The stake option despite appearing on the screen it doesn't seem to be working, at least on iOS"

**Comment by TJ-BitCanna** [comment]
@JohnnyX89 
This is currently being developed and was indeed not live in the current released test version.

**Comment by arcturiano** [comment]
Was just going to add this as an issue, stake button not enabled yet! :)

**Reply by arcturiano** [reply]
I see it, but it does nothing when cicked
